packages/react-components/src/Status/index.tsx

```diff
@@ -144,8 +145,24 @@ function Status ({ className, stqueue = [], txqueue = [] }: Props): React.ReactE
     return null;
   }
 
+  const _onDismiss = (): void => {
+    allst.map(s => s.removeItem());
+    alltx.map(t => t.removeItem());
+  };
+
   return (
     <div className={`ui--Status ${className}`}>
+      {allst.length + alltx.length > 1 && (
+        <div className='dismiss'>
+          <Button
+            isNegative
+            onClick={_onDismiss}
+            label={t('Dismiss all')}
+            icon=''
```

Comment:
Add the `cancel` as an icon. (It is left aligned which is a bit unfortunately, however, it is fine in this case)

Reply:
Would also suggest having the button stretch across the full notification width.